themes/src/main/resources/theme/keycloak.v2/account/src/app/content/linked-accounts-page/LinkedAccountsPage.tsx

```diff
@@ -185,9 +179,22 @@ class LinkedAccountsPage extends React.Component<LinkedAccountsPageProps, Linked
                         <DataListItemRow key={account.providerName}>
                             <DataListItemCells
                                 dataListCells={[
-                                    <DataListCell key='idp'><Stack><StackItem isFilled>{this.findIcon(account)}</StackItem><StackItem id={`${account.providerAlias}-idp-name`} isFilled><h2><strong>{account.displayName}</strong></h2></StackItem></Stack></DataListCell>,
-                                    <DataListCell key='label'><Stack><StackItem isFilled/><StackItem id={`${account.providerAlias}-idp-label`} isFilled>{this.label(account)}</StackItem></Stack></DataListCell>,
-                                    <DataListCell key='username' width={5}><Stack><StackItem isFilled/><StackItem id={`${account.providerAlias}-idp-username`} isFilled>{account.linkedUsername}</StackItem></Stack></DataListCell>,
+                                    <DataListCell key='idp'>
+                                        <Split>
+                                            <SplitItem className="pf-u-mr-sm">{this.findIcon(account)}</SplitItem>
+                                            <SplitItem className="pf-u-my-xs" isFilled><span id={`${account.providerAlias}-idp-name`}>{account.displayName}</span></SplitItem>
+                                        </Split>
+                                    </DataListCell>,
+                                    <DataListCell key='label'>
+                                        <Split>
+                                            <SplitItem className="pf-u-my-xs" isFilled><span id={`${account.providerAlias}-idp-label`}>{this.label(account)}</span></SplitItem>
+                                        </Split>
+                                    </DataListCell>,
+                                    <DataListCell key='username' width={5}>
+                                        <Split>
+                                            <SplitItem className="pf-u-my-xs" isFilled><span id={`${account.providerAlias}-idp-username`}>{account.linkedUsername}</span></SplitItem>
+                                        </Split>
+                                    </DataListCell>,
                                 ]}/>
                             <DataListAction aria-labelledby='foo' aria-label='foo action' id='setPasswordAction'>
```

Comment:
Do you still want "foo" here?

Reply:
I've updated all references of foo to have a more context-fitting name